clarity-ext-scripts/deployment/get-slice-tool.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Usage: ./get-slice-tool <SERVERNAME>
```

Comment:
I guess you actually need both user and server name here?

Reply:
Right, unless the username is the same.

Reply:
Fixed.